docs/services/Linkedin.md

```diff
@@ -63,7 +63,7 @@ Await LinkedInService.Instance.GetUserProfileAsync()
 Await LinkedInService.Instance.ShareActivityAsync(ShareText.Text)
 ```
 
-[LinkedIn Service Sample Page Source](https://github.com/windows-toolkit/WindowsCommunityToolkit/tree/rel/7.0.0/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/LinkedIn%20Service). You can see this in action in [Windows Community Toolkit Sample App](https://www.microsoft.com/store/apps/9NBLGGH4TLCQ).
+[LinkedIn Service Sample Page Source](https://github.com/windows-toolkit/WindowsCommunityToolkit/tree/rel/7.1.0/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/LinkedIn%20Service). You can see this in action in [Windows Community Toolkit Sample App](https://www.microsoft.com/store/apps/9NBLGGH4TLCQ).
```

Comment:
Looks like some of these old deprecated things got caught in the last mass update, we should either point them to the historical code or remove them. We can create an issue to fix later.